main.tf

```diff
@@ -60,3 +60,28 @@ resource "azurerm_automation_account" "pre-aa" {
 
   tags = var.common_tags
 }
+
+data "azurerm_key_vault_secret" "slack_monitoring_address" {
+  name         = "slack-monitoring-address"
+  key_vault_id = data.azurerm_key_vault.keyvault.id
+}
+
+resource "azurerm_application_insights" "this" {
```

Comment:
although it's not referenced else where, would it be better to name this `"ai"` or something?

Reply:
its a convention used in the terraform best practices guide https://www.terraform-best-practices.com/naming and so if there isn't a more descriptive name for the resource, "this" is preferred

Reply:
TIL - thanks